### PR TITLE
Add MOTD

### PR DESCRIPTION
**Pull request by Wilma456**
Simple add a MOTD, who shows the Newbies some basic things. We can add more texts in the future.

### CODE REVIEW

**Comment by SquidDev** [comment]
Ooooh, this is a really nice idea. Good shout having settings to disable it & configure where it looks too. Just a couple of thoughts:

 - I'd almost be tempted to have the settings under `startup.*` (or even `motd.*`) instead. I know we've got `shell.enable_disk_startup`, but at the same time it does feel a little bit weird to keep putting startup related things onto the shell.
 - I wonder if it'd be worth having a `motd` program, which does all the displaying, and then the startup file just `shell.run`/`dofile`s that? This means people with custom shells can also display it without duplicating code.
 - I think we need to have a think about how this is presented. Dan has always stressed that new users should have to deal/see as little as possible (hence `edit` being overly simplistic), so I don't want to confuse people with lots of extra text. But at the same time, this could be a very useful tool.
 - Probably worth everyone brainstorming some things which could be added to the MOTD. I don't think links to the GH or Forum are incredibly useful for users, but there's lots of other things we could point out (say, talking about the `peripherals` or `rs` programs, etc...)

Sorry for the wall of text - it's definitely a really nice PR (<3), but it's also something which affects everyone quite visibly, so I want to tread carefully.

**Comment by JasonTheKitten** [comment]
"this texts" on the .txt file is a typo - should be "these texts"

**Comment by Wilma456** [comment]
I had made some changes suggested by @SquidDev . I think we should keep the links to the websites, so new User directly knows where to get more Information. If the Player uses Google, they will find the old computercraft.info Website, which is outdated. If you tell me more Messages, I will add it to motd.txt.

**Comment by kepler155c** [comment]
Perhaps point them to help files instead: ie. help forums, or help docs.

**Comment by Lemmmy** [comment]
How about we just ensure all the relevant links (wiki, forums, GitHub, issue tracker) are available on https://computercraft.cc, and then just link to that?

**Comment by Wilma456** [comment]
I think, we should have a extra link for each option.

What is the opinion of @SquidDev?

File: src/main/resources/assets/computercraft/lua/rom/motd.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+View the source code at https://github.com/SquidDev-CC/CC-Tweaked
+View the documentation at https://wiki.computercraft.cc
+Visit the forum at https://forums.computercraft.cc
+You can disable these texts by running "set motd.enable false"
```
Comment (SquidDev) [inline src/main/resources/assets/computercraft/lua/rom/motd.txt]:
```suggestion
You can disable these messages by running "set motd.enable false"
```

Just makes a little bit more sense :)

File: src/main/resources/assets/computercraft/lua/bios.lua [hunk 2]
```diff
@@ -982,6 +982,8 @@ settings.set( "edit.default_extension", "lua" )
 settings.set( "paint.default_extension", "nfp" )
 settings.set( "lua.autocomplete", true )
 settings.set( "list.show_hidden", false )
+settings.set( "motd.enable", true )
```
[the remark this answers is not in the document]
Comment (Wilma456) [reply src/main/resources/assets/computercraft/lua/bios.lua]: I think it should be enabled by default. The MOTD is mostly for newbies which used CC-T for the first time or discovered in a modpack. If they know how too enable it, they are no newbies anymore.

[the remark this answers is not in the document]
Comment (SquidDev) [reply src/main/resources/assets/computercraft/lua/bios.lua]: I'm not disagreeing with that, I'd just like to get the message list a little larger + a little more useful before turning it on by default.

**Comment by Wilma456** [comment]
@SquidDev The Changes have now been done

**Pull request merged**